### PR TITLE
ch(setup-react-project): Setup an empty react project

### PR DESCRIPTION
**Pull request by Francoise02**
#### What does this PR do?

Set up an empty react project using Webpack 4, Babel, and the Airbnb linting option.

#### How should this be manually tested?

1. Clone branch, install packages by running or `yarn `.
2. Run `yarn start` to start the server. 
3. Now you can see a page with the **Barefoot Nomad** title opened in your browser.
4. To test linting, run `yarn lint` and fix the lining errors run `yarn lint:fix`

#### What are the relevant pivotal tracker stories?
[#182201515](https://www.pivotaltracker.com/story/show/182201515)

### CODE REVIEW

**Review by Julienish14 — APPROVED** [review]

**Review by ishimweeli — APPROVED** [review]

**Review by kalisaNkevin — APPROVED** [review]

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  render() {
+    return (
+        <div>
+           <h1> Barefoot Nomad</h1>
+        </div>
+    );
+  }
+}
+
+export default App;
```
Comment (ericus123) [inline src/components/App.js]: Use function based components other than class based.
They work better and are more updated.

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+        },
+      },
+    ],
+  },
```
Comment (ericus123) [inline webpack.config.js]: try adding some other common loaders configurations here as well.
Like style loaders(css,scss) and images loaders (png , svg, jpg , ...).
with the current loaders configuration styles and images can't be loaded by webpack.

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+};
+
+{
+  'production'; // | 'development' | 'none'
+}
```
Comment (ericus123) [inline webpack.config.js]: what's the use of this?
remove it if not used

File: package.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
+{
+  "name": "elite-bn-fe",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server --mode development --open --hot",
+    "build": "webpack --mode production",
+    "lint": "eslint src/**/*.js",
+    "lintFix": "eslint --fix --ext src/**/*.js",
```
Comment (ericus123) [inline package.json]: don't use camelCase for scripts.
you can do this instead. `lint:fix`

**Review by ericus123 — CHANGES_REQUESTED** [review]

**Review by ericus123 — APPROVED** [review]

**Pull request merged**